interactive_ai/services/resource/pyproject.toml

```diff
@@ -9,6 +9,7 @@ dependencies = [
     "jsonschema>=4.18.0a10",
     "python-multipart~=0.0.18",
     "uvicorn~=0.27",
+    "pillow==11.1.0",
```

Comment:
Now that it's a runtime dependency, can you remove it from dev dependencies (L47)?